changelog.d/20220810_120055_nicola.coretti__106_Support_Sqlalchemy_1_4.rst

```diff
@@ -0,0 +1,10 @@
+🔧 Changed
+----------
+- Update SQLAlchemy dependency to >= 1.4
+
+    .. warning::
+
+        This may impact the performance (see also `SQLAlchemy docs <https://docs.sqlalchemy.org/en/14/faq/performance.html#why-is-my-application-slow-after-upgrading-to-1-4-and-or-2-x>`_).
```

Comment:
If, that means probably quite a lot work

Reply:
Do you know, if it at least caches the metadata queries?

Reply:
I don't no yet. Because the caching topic seems to be quite complex I moved it into an follow up issue, so we can focus on the migration to 1.4 first.